tests/lora/utils.py

```diff
@@ -244,7 +247,7 @@ def test_simple_inference_with_text_lora(self):
                         check_if_lora_correctly_set(pipe.text_encoder_2), "Lora not correctly set in text encoder 2"
                     )
 
-            output_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
+            output_lora = pipe(**inputs, generator=torch.manual_seed(0))[0]
```

Comment:
ohh do we need to update the `get_dummy_inputs` to set `return_dict = False`?

Reply:
I don't think `return_dict=False` required as we do this in many of our tests actually. Deriving from BaseOutput and decorating with `@dataclass` makes it possible to index outputs. The `__getitem__` of BaseOutput converts the underlying attribute dict to tuple and indexes it IIUC

```
from dataclasses import dataclass
from diffusers.utils import BaseOutput

@dataclass
class Test(BaseOutput):
    value1: str
    value2: str

test = Test("hello", "world")
print(test[0])
print(test[1])
```

Reply:
ok good to know!